deploy/sample-backup-configs/sample_config_runtime.yaml

```diff
@@ -1,17 +1,7 @@
 # RUNTIME configuration for the FireSim Simulation Manager
 # See https://docs.fires.im/en/stable/Advanced-Usage/Manager/Manager-Configuration-Files.html for documentation of all of these params.
 
-run_farm:
-    run_farm_tag: mainrunfarm
-    always_expand_runfarm: yes
-    f1_16xlarges: 1
-    m4_16xlarges: 0
-    f1_4xlarges: 0
-    f1_2xlarges: 0
-    launch_instances_timeout_minutes: 60
-    run_instance_market: ondemand
-    spot_interruption_behavior: terminate
-    spot_max_price: ondemand
+run_farm: aws_ec2_f1_run_farm
```

Comment:
I think this is something that we might want to discuss in more detail. IMO once you start adding overrides it makes it confusing to find out what values are used. Here are a couple of schemas that might work:

1. Have the run farm configuration inlined (in addition to the ability to put it in a different file):

```yaml
# inlined case
run_farm:
  run_farm_name:
  run_farm_type: XYZ
  args:
    ....
    
# non-inlined case
run_farm: run_farm_special_name

# or we could keep it uniform
run_farm:
  run_farm_name: run_farm_special_name
```

2. Specify an override dict - like what you want. Basically it will override an argument given. To override you would have to do something like this?

```yaml
run_farm:
  name: XYZ
  overrides:
    run_farm_tag: MY OVERRIDE
    run_farm_hosts:
       - f1.2xlarge: 20
```

This problem also exists for the build farms as well (mainly for the externally provisioned case).

I think this gets tricky if there are nested dicts. For example, should `run_farm_hosts` keep the values of the other F1 instances and only change `f1.2xlarge`? Or should you have to re-specify all the F1 instances (respecify the count of `m4.16`s etc (in the current example the `m4.16` would be 0 by default as well as the other F1 instances).